manifests/n/Nvidia/GeForceExperience/3.25.0.84/Nvidia.GeForceExperience.installer.yaml

```diff
@@ -0,0 +1,26 @@
+# Created with YamlCreate.ps1 v2.0.6 $debug=QUSU.7-2-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Nvidia.GeForceExperience
+PackageVersion: 3.25.0.84
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallerType: exe
+Scope: machine
+InstallModes:
+- interactive
+- silent
+InstallerSwitches:
+  Silent: -s
+  SilentWithProgress: -s
+  Custom: -noreboot
+InstallerSuccessCodes:
+- 3858759680
```

Comment:
This exit codes solves most Nvidia software issues as it means no Nvidia GPU found